### ISSUE TITLE
Duplicate entries

### ISSUE THREAD

**Issue by nicksavov** [OSTraining/OSMap#37]
See:
https://www.alledia.com/blog/osmap-30/#comment-1904953934

**Comment by andergmartins** [comment]
Right, I will check, run the migration tool twice would explain that. The v2.4 wasn't removing the Xmap data after migrating, but v3.0 removes. Maybe after install the v3 he saw the migration button again, and migrated the data again (since we don't touch current data without the user action) because the older version didn't wiped it.

But I will double check making some tests
